com.stansassets.build/Editor/Steps/ExecuteFinishedArgs.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace StansAssets.Build.Editor
+{
+    public class ExecuteFinishedArgs
```

Comment:
Some fields are missing in this class. How can I know if the build step has been finished successfully?
What about string ErrorMessage if step failed?

Reply:
I don't really like this mode. Let's talk more about it